jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -104,26 +111,43 @@ pipeline {
                     }
                     echo "doPush = ${doPush}"
 
-                    if (doPush) {
-                        gitPushBranch(productBranch, true)
+                    if (dryRun) {
```

Comment:
We actually don't need to manage the dryRun here as it's covered by https://github.com/integr8ly/delorean-pipeline-library/blob/master/vars/gitPushBranch.groovy#L12 TODO: Remove this.